templates/main.html

```diff
@@ -45,49 +56,48 @@ <h3>
                 {{name}}
               </a>
             </h3>
-            <p>Tasks Completed: {{completed_task_instance_count}}</p>
+            <p><span data-i18n="tasks-completed">Tasks Completed</span>: {{completed_task_instance_count}}</p>
             {{#github}}
               <a href="https://github.com/{{github}}">
                 <img
                   src="https://assets-cdn.github.com/images/modules/logos_page/GitHub-Mark.png"
                   class="github"
-                  alt="Find them on GitHub"
                 />
               </a>
             {{/github}}
             {{#mailing_list}}
               <a href="{{mailing_list}}">
-                <img src="images/mail.png" class="chat" alt="Join their mailing list" />
+                <img src="images/mail.png" class="chat" />
```

Comment:
why removing all the `alt`?
`jquery.i18n` doesnt support them?
English only `alt` is better than no `alt`..?

Reply:
may also be related to the #78 merge